common/lib/common-utils/src/base64Encoding.ts

```diff
@@ -5,8 +5,16 @@
 
 import { IsoBuffer } from "./indexNode";
 
+/**
+ * Converts the provided {@link https://en.wikipedia.org/wiki/Base64 | base64}-encoded string
```

Comment:
Not for this PR, but I wonder whether we should have lint rules for comment to make these term are consistently linked?

Reply:
That's a great idea!